github/ci/prow-deploy/files/jobs/kubevirt/containerdisks/containerdisks-presubmits.yaml

```diff
@@ -88,7 +88,7 @@ presubmits:
       nodeSelector:
         type: bare-metal-external
   - always_run: true
-    optional: true
+    optional: false
```

Comment:
@nestoracunablanco just a nit comment can we be consistent here? for the rest of the jobs you have just removed `optional: true`- can you do the same here?